Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM python:2.7.12
+
+ARG BUILD_DEPS=" \
+    build-essential \
+    git \
+    libffi-dev \
+    libssl-dev \
+    libxml2-dev \
+    libxslt1-dev"
+
+RUN apt-get update && \
+    apt-get install \
+        --no-install-suggests \
+        --no-install-recommends \
+        -y \
+        $BUILD_DEPS && \
+    rm -rf /var/lib/apt/lists/*
+
+# Node
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
+RUN apt-get install -y nodejs
+
+RUN mkdir /code
+RUN mkdir /code/website
+RUN mkdir /code/plugins
+WORKDIR /code/website
+
+# Python dependencies
+RUN pip install uWSGI==2.0.13.1
+COPY ./requirements.txt /code/website
+RUN pip install -r requirements.txt
+
+# Node dependencies
+COPY ./package.json /code/website
+RUN npm install
+
+COPY . /code/website
+
+# Translations
+RUN pybabel compile -d website/frontend/translations
+
+# Static files
+RUN ./node_modules/.bin/gulp
+
+# Plugins
+RUN git clone https://github.com/metabrainz/picard-plugins.git /code/plugins
+WORKDIR /code/plugins
+RUN python generate.py
```

Comment:
Wouldn't work for now because https://github.com/metabrainz/picard-plugins/blob/master/generate.py#L96.
```
Step 19 : RUN python /code/plugins/generate.py
 ---> Running in 87cb082139f6
Traceback (most recent call last):
  File "/code/plugins/generate.py", line 108, in <module>
    build_json()
  File "/code/plugins/generate.py", line 26, in build_json
    for dirname in next(os.walk(plugin_dir))[1]:
StopIteration
```

Should be relative to `os.path.realpath(__file__)`.